event-driven-architecture/index.md

```diff
@@ -0,0 +1,33 @@
+layout: pattern
+title: Event Driven Architecture
+folder: event-driven-architecture
+permalink: /patterns/event-driven-architecture
+
+
+**Intent:** Send and notify state changes of your objects to other applications using an Event-driven Architecture.
+
+![alt text](./etc/class_diagram.png "Event Driven Architecture")
+
+**Applicability:** Use an Event-driven architecture when
+
+* you want to create a loosely coupled system
+* you want to build a more responsive system
+* you want a system that is easier to extend
+
+**Real world examples:**
+
```

Comment:
It will be great if you can link any actual applications as well. Any existing pattern or library that uses this architecture.